src/ccs/confluent.sh

```diff
@@ -1707,6 +1719,7 @@ connect_subcommands() {
 }
 
 version_command() {
+    validate_os_version "${1}"
```

Comment:
I know we talked about adding the check to the version command, but it seems that the script will die and thus it never proceeds to print the CP version.  This is undesirable.  Perhaps let's just remove the check from version and keep it with the start command?  That way it'll also be consistent with the java check

Reply:
Adding `validate_os_version "${1}"` at the end of this method achieves both purposes. 
prints both the confluent platform's version (which is unaffected by this issue with rocksdb version) and also prints the error message. I suggest keeping but moving as last call in the `version_command` function.

Reply:
I think @ybyzek said there's a lot of automation that depends on the last line printed being the version -- so probably safer to just remove this.  I'll remove it for now but happy to re-add if there's consensus!

Reply:
Still, I think it's safe to leave in the end of this function. Because even if automation is present, then it depends on a) a version without the problem, in which case no error message is printed, or b) the error is printed on a system for which some services won't work anyways. 

Don't feel too strong about it though. `confluent version` is not as frequent as just `confluent` or `confluent help`